src/apps/api/urls.py

```diff
@@ -43,7 +45,7 @@
     url(r'swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
     path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
     path('redoc/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
-
+    path('can_make_submission/<phase_id>/', submissions.can_make_submission),
```

Comment:
So thinking about this more, can we easily do this step on getting the `SAS` url? Hmmm....